src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -420,4 +421,52 @@ public void setNotFailingStatuses_SkipsNullValues() {
         // then
         assertThat(configuration.getNotFailingStatuses()).containsExactly(notFailingStatus);
     }
+
+    @Test
+    public void chartsNotToRender_FEATURES_STATISTICS() {
```

Comment:
I think that Sonar will report naming convention violation :)

Reply:
I assume the violation would be due to the uppercasing versus camel-casing?

Reply:
yes

Reply:
also try to use naming like for other methods: methodName_expectation()

Reply:
Renamed appropriately